app.py

```diff
@@ -9,6 +9,9 @@
 This app is designed to be the dev and production deployment app.
 It defaults to a dev deployment via a default `env` value in cdk.json.
 To deploy to prod, specify `--context env=prod`.
+To deploy to the backup account (only deploys required backup stacks),
+specify `--context env=backup`.
+
```

Comment:
It's not related to your PR but wanted to mention that at one point, we need to update this `app.py` to match `app_template_dev.py`. Also, I don't know when we use `app.py` vs `app_template_dev.py`. I will create ticket for this task.

Reply:
I wasn't clear what parts of app.py needed to be updated to match with app_template_dev.py so I left it as is. I'm happy to copy over app_template_dev.py and remove all the initial specific stuff if that's preferable!

Reply:
me too. I created a ticket for this https://github.com/IMAP-Science-Operations-Center/sds-data-manager/issues/127